code/index.html

```diff
@@ -1,33 +1,31 @@
 <!DOCTYPE html>
 <html lang="en">
 
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./style.css" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
-      rel="stylesheet" />
-    <title>Chat Bot</title>
-  </head>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./style.css" />
+  <title>Happy developer-bot!</title>
+</head>
 
-  <body>
-    <h1>Welcome to xxx.</h1>
-    <main>
-      <section class="chat" id="chat"></section>
+<body>
+  <h1>Welcome to the Happy developer-bot!</h1>
+  <h2>It's riddle time! </h2>
```

Comment:
It quickly determine the intent of your content when using both h1 and h2 which is great for the SEO!